RecipeMatch/Views/DraggableViewBackground.m

```diff
@@ -18,6 +18,7 @@ @interface DraggableViewBackground ()
 @implementation DraggableViewBackground{
     NSInteger cardsLoadedIndex; // the index of the last card loaded into the loadedCards array
     NSMutableArray *loadedCards; // the array of card loaded
+    NSInteger allCardsIndex; // index of current card out of all cards
```

Comment:
i find the naming of `cardsLoadedIndex ` and `allCardsIndex ` to be little confusing....any chance you can comeout with a better naming?